spec/spec_helper.rb

```diff
@@ -13,6 +15,19 @@
 # it.
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+require 'simplecov'
```

Comment:
We've got Slayer as a dependency, we should also setup groups for the various slayer modules: `Slayer::Commands` `Slayer::Services` `Slayer::Forms`